metadata-ingestion/tests/integration/lookml/test_lookml.py

```diff
@@ -1029,19 +1030,16 @@ def test_gms_schema_resolution(pytestconfig, tmp_path, mock_time):
         "my_connection": "hive"
     }
 
-    with patch(
-        "datahub.ingestion.source.looker.view_upstream.SchemaResolver"
-    ) as MockSchemaResolver:
-        mock_instance = MockSchemaResolver.return_value
-        mock_instance.__enter__.return_value = mock_instance
-        mock_instance.resolve_urn.return_value = (
-            "urn:li:dataset:(urn:li:dataPlatform:dbt, db.public.employee, PROD)",
-            {
-                "Id": "String",
-                "Name": "String",
-                "source": "String",
-            },
-        )
+    return_value: Tuple[str, Optional[SchemaInfo]] = (
+        "urn:li:dataset:(urn:li:dataPlatform:dbt, db.public.employee, PROD)",
```

Comment:
why are there spaces in the returned urn?

Reply:
actually why is this a dbt urn - is this urn even being used anywhere?

Reply:
it is fake, I updated it